modules/op-running-a-pipeline.adoc

```diff
@@ -0,0 +1,65 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+
+[id="running-a-pipeline_{context}"]
+= Running a Pipeline
+
+A PipelineRun starts a Pipeline and ties it to the Git and image resources that should be used for the specific invocation. It automatically creates and starts the TaskRuns for each Task in the Pipeline.
+
+[discrete]
+.Procedure
+
+. Start the Pipeline for the back-end application:
++
+----
+$ tkn pipeline start build-and-deploy -r git-repo=api-repo -r image=api-image -p deployment-name=vote-api
+----
++
+Note the PipelineRun ID returned in the command output.
+
+. Track the PipelineRun progress:
++
+----
+$ tkn pipelinerun logs <pipelinerun ID> -f
+----
+. Start the Pipeline for the front-end application:
++
+----
+$ tkn pipeline start build-and-deploy -r git-repo=ui-repo -r image=ui-image -p deployment-name=vote-ui
+----
++
+Note the PipelineRun ID returned in the command output.
+
+. Track the PipelineRun progress:
++
+----
+$ tkn pipelinerun logs <pipelinerun ID> -f
+----
+. After a few minutes, use `tkn pipelinerun list` command to verify that the Pipeline ran successfully by listing all the PipelineRuns.
++
+----
+$ tkn pipelinerun list
+----
++
+The output lists the PipelineRuns.
```

Comment:
Suggest being consistent with colon or period introducing a copy block. Based on usage in this doc, it makes the most sense that this change to a colon.